src/style.css

```diff
@@ -118,7 +118,7 @@ input[type='checkbox']:checked {
   outline: none;
   margin: 0;
   padding: 0;
-  color: pink;
+  color: #7c73e6;
```

Comment:
- [ ] Hello perhaps because of the way you arrange your folders I am unable to make an inline comment on your files js files in your src this is why I make my comment here 
- [ ] Great job in implementing the check box but there are still issues you still need to address to make your project complete.
- [ ] I would suggest you to try as much as posible to adhere to the project requirement ,your clear all completed button is not functioning as expected  the requirement says  `Implement a function for the "Clear all completed" button (use filter() method).
Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved)`kindly fix this so you todolist will look professional .

Reply:
Dear @ademibowale 👋🏾
Thank you for your kind suggestions, 

1) In my file there is the filter method placed in removeAllCompletedTask , 
please can you see this screenshot: (here is placed the filter method)

<img width="1177" alt="filterFunction" src="https://github.com/IvonneBenitesRodriguez/todo-list-microverse/assets/22691013/a8474e59-4bce-4817-bcc8-3d45e6bfe701">

2) Besides, the data is stored in the localStorage please see this screenshot:

<img width="1109" alt="localstorage" src="https://github.com/IvonneBenitesRodriguez/todo-list-microverse/assets/22691013/7fbefb52-468d-45a3-a886-e6fbf3615d3f">
It helps me to preserve data in the localstorage.

Reply:
> Dear @ademibowale 👋🏾 Thank you for your kind suggestions,
> 
> 1. In my file there is the filter method placed in removeAllCompletedTask ,
>    please can you see this screenshot: (here is placed the filter method)
> 
> <img alt="filterFunction" width="1177" src="https://user-images.githubusercontent.com/22691013/265225159-a8474e59-4bce-4817-bcc8-3d45e6bfe701.png">
> 2. Besides, the data is stored in the localStorage please see this screenshot:
> 
> <img alt="localstorage" width="1109" src="https://user-images.githubusercontent.com/22691013/265225883-7fbefb52-468d-45a3-a886-e6fbf3615d3f.png"> It helps me to preserve data in the localstorage.

I am sorry for the late reply please, I understand but try to re-check your project and cross-check the project requirement when the user clicks all the check boxes the user can delete or clear all items completed at once.